### PR TITLE
Ped bug fix

### LINKED ISSUE #128: [PE-D] Bug in UG link to software `jar` file
Link to FoodOnWheel's `jar` file does not take me to the releases page on GitHub. Consider changing the link to make it easier for the user to install your software.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: aditi2313/ped#3

### LINKED ISSUE #130: [PE-D] Bug in UG Quick start to render GUI
Sample GUI to be rendered when the software is started seems to have a broken link. 

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: aditi2313/ped#4

### LINKED ISSUE #137: [PE-D] The URL for the user guide image of the `HelpCommandWindow` is not updated
-------------
Labels: `severity.High` `type.DocumentationBug`
original: s7manth/ped#6

### LINKED ISSUE #141: [PE-D] Formatting issue with link to Features
“Refer to features below” should not be a bullet point. Recommend changing its format so the user can see it more clearly.

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: aditi2313/ped#9

### LINKED ISSUE #144: [PE-D] Buggy command in example commands relating to `Order`
List of sample `Order` commands to try (more specifically `addorder p/98765432 d/Sushi, Pasta`) seems to be an invalid command when you try running it right after starting the app. Ideally your example commands section should only list out commands that can be copy pasted into the command box and should work regardless of the order they are executed in. Consider replacing the parameters of this command with others.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: aditi2313/ped#10

### LINKED ISSUE #151: [PE-D] Date time formatting error
-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: ongkimlai/ped#11

### LINKED ISSUE #153: [PE-D] Edit same phone number does not have error when editing phone only
[Screen Recording 2022-04-01 at 2.56.28 PM.mov](https://raw.githubusercontent.com/Denniszedead/ped/main/files/f353b5fa-ca5a-4c01-a6b7-358371453cbb.mov)

When editing name and phone together, with the same phone number, it returns an error. However, when edit only phone number, it succeeds.

It occurs only in the customer section.

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: Denniszedead/ped#5

### LINKED ISSUE #154: [PE-D] Can edit same name
[Screen Recording 2022-04-01 at 3.00.36 PM.mov](https://raw.githubusercontent.com/Denniszedead/ped/main/files/ef76a55f-98c6-45c0-8f63-22b3b19c0376.mov)

Able to edit same name but by default when adding customer no duplicated name allowed.

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: Denniszedead/ped#6

### PR DESCRIPTION
**Pull request by xzzz3**
Fixes #128 fixes #137 fixes #141 fixes #144 fixes #151 fixes #153 fixes #154 

Also fixes #130 since I tested on my end and the GUI renders correctly

### CODE REVIEW

**Review by zxgoh — APPROVED** [review]
LGTM, thanks!

File: src/main/java/seedu/address/model/order/Order.java [hunk 1]
```diff
@@ -130,7 +130,7 @@ public String toString() {
                 .append("; Status: ")
                 .append(getStatus())
                 .append("; Time: ")
-                .append(getTime());
+                .append(getTime().toString().replace("T", " "));
```
[the remark this answers is not in the document]
Comment (xzzz3) [reply src/main/java/seedu/address/model/order/Order.java:133]: this replaces the "T" in the default datetime tostring with a whitespace, so now its something like 
image

[the remark this answers is not in the document]
Comment (xzzz3) [reply src/main/java/seedu/address/model/order/Order.java:133]: just pushed a new commit to make the string look like this (removed the decimal)
image

[the remark this answers is not in the document]
Comment (hieunm1821) [reply src/main/java/seedu/address/model/order/Order.java]: Looks nice!

**Review by hieunm1821 — APPROVED** [review]

**Pull request merged**